WalletWasabi.Packager/Program.cs

```diff
@@ -61,7 +55,7 @@ public static class Program
 		/// <summary>
 		/// Main entry point.
 		/// </summary>
-		private static void Main(string[] args)
+		private static async Task Main(string[] args)
```

Comment:
```suggestion
#pragma warning disable IDE1006 // Naming Styles

		private static async Task Main(string[] args)
#pragma warning restore IDE1006 // Naming Styles
```

Reply:
Isn't there a nicer solution? Those comments do the job but it looks kind of ugly. :)

Reply:
I am not aware of a nicer solution.